src/main/java/sort/NaturalSort.java

```diff
@@ -0,0 +1,37 @@
+package sort;
+
+import org.neo4j.procedure.Name;
+import org.neo4j.graphdb.Node;
+import org.neo4j.procedure.Procedure;
+import org.neo4j.procedure.UserFunction;
+import sort.*;
+import java.util.*;
+
+public class NaturalSort
+{
+    @UserFunction
+    // @Description("sort.natural_list(coll) sort on primitive collection")
+    public List<Object> natural_list(
+        @Name("coll") List<Object> coll
```

Comment:
IMO full variable name will be more transparent `collection` or `list`. Maybe it's just me, but coll is easy to misunderstood like `col` (column)

Reply:
Agree 👍 thanks for pointing it out!